src/Features/LanguageServer/Protocol/Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.LanguageServer</RootNamespace>
+    <CreateVsixContainer>false</CreateVsixContainer>
```

Comment:
><CreateVsixContainer>false</CreateVsixContainer> [](start = 4, length = 48)

Is this necessary? A VSIX shouldn't be created unless the directory contains VSIX manifest file.